offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/create/CreateAffiliationView.groovy

```diff
@@ -25,385 +23,106 @@ import java.util.stream.Collectors
  * @since: 1.0.0
  */
 @Log4j2
-class CreateAffiliationView extends FormLayout {
+class CreateAffiliationView extends FormLayout implements Resettable, SubmitNotifier, AbortNotifier {
     final public AppViewModel sharedViewModel
-    final public CreateAffiliationViewModel createAffiliationViewModel
     private final CreateAffiliationController controller
+    private final ResourcesService<Affiliation> affiliationResourcesService
 
-    private ComboBox<String> organisationBox
-    private TextField addressAdditionField
-    private TextField streetField
-    private TextField postalCodeField
-    private TextField cityField
-    private ComboBox<String> countryBox
-    private ComboBox<String> affiliationCategoryField
+    private List<SubmitListener> submitListeners = new ArrayList<>()
+    private List<AbortListener> abortListeners = new ArrayList<>()
```

Comment:
Nit: This does not have to be a list currently since we only have one submit and abort button for each view.